less/misc.less

```diff
@@ -53,6 +53,9 @@ input, textarea {
 	visibility: hidden;
 	top: 0;
 	float: left;
+	.selectlist {
+		display: block !important;
```

Comment:
This is just for the dummy selectlist I create. The element I append to the body to get the width of the dropdown items then remove once I have a width.